kurtosis-core-rpc-api/api_container_service.proto

```diff
@@ -152,6 +152,9 @@ message RegisterServiceArgs {
 message RegisterServiceResponse {
   // The IP address that the service will receive when it starts
   string ip_addr = 1;
+
+  // The relative path from the container vol root to the unique service directory in the enclave data volume
```

Comment:
container root actually isn't a part of this - recommend rewording to "The dirpath where the service's directory exists inside the enclave data volume, relative to the enclave data volume root"